site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
+
+Chrome 91 adds the following features to the extensions platform.
+
+#### ES modules for service workers
+
+JavaScript supports modules in service workers. Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts. (https://www.chromestatus.com/guide/edit/4609574738853888)
+
+#### chrome.action.getUserSettings() available
+
+The new chrome.action.getUserSettings() API allows extensions to determine if the user has pinned the extension to the main toolbar.
+
+### Chrome 90 (stable on April 13, 2021)
+
+#### New chrome.scripting.removeCSS() method (MV3 only)
+
+The `chrome.scripting.removeCSS()` method lets extensions remove CSS that was previously inserted via
+chrome.scripting.insertCSS(). It replaces the MV2 method `chrome.tabs.removeCSS()`.
+
+#### chrome.scripting.executeScript() results now include frameId
+
+Results returned from `chrome.scripting.executeScript()` now include the frameId.  The frameId indicates the frame from which the result is from, allowing extensions to easily associate results with the individual frames when injecting in multiple frames.
+
+### April 8, 2021: Extension Manifest Converter
+
+The Chrome Extensions team has open sourced [Extension Manifest
+Converter](https://github.com/GoogleChromeLabs/extension-manifest-converter), a Python tool that
+helps to convert extensions to Manifest V3. 
+
+### Chrome 89 (stable on March 9, 2021)
+
+#### New chrome.tabGroups API
+
+Added the chrome.tabGroups API to let extensions read and manipulate tab groups. Manifest V3 only.
+
+#### Customizable permissions for MV3 Web Accessible Resources
+
+Manifest now supports permission definitions to restrict resources based on either the accessing site origin or extension ID.https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/
```

Comment:
phrasing sounds a little abrupt here.  (At the very least, "The manifest", but perhaps better would be "You can now specify..." or "Web-accessible resources can be...", etc?)  Also linkify text, rather than drop the plaintext link?

Reply:
```suggestion
[Web accessible resources][war] definitions in Manifest V3 have changed in order to allow extensions to restrict resource access based on the requester's origin or extension ID. 

[war]: https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/
```